utils/supabase/supabase.ts

```
@@ -0,0 +1,65 @@
+    process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!
+  )
+}
+
+/**
+ * Asynchronously creates and returns a Supabase client configured for use in Next.js server components, with cookie management integrated via the Next.js cookie store.
+ */
+export async function createServerComponentClient() {
+  const cookieStore = await cookies()
+
+  return createServerClient(
+    process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)